Source/relay/TourGuide/Items of the Month/2023/SIT Course Certificate.ash

```diff
@@ -0,0 +1,35 @@
+boolean hasAnySkillOf(string [int] skillNames) {
+    foreach i in skillNames {
+        string skillName = skillNames[i];
+        if (lookupSkill(skillName).have_skill()) {
+            return true;
+        }
+    }
+    return false;
+}
+
+// Prompt to register which SIT course you took
+RegisterTaskGenerationFunction("IOTMSITCertificateGenerateTasks");
+void IOTMSITCertificateGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries) {
+    if (!lookupItem("S.I.T. Course Completion Certificate").have())
+        return;
+
+    // Nag if we haven't picked a skill during this ascension    
+    string [int] skillNames = {"Psychogeologist", "Insectologist", "Cryptobotanist"};
+    if (hasAnySkillOf(skillNames)) {
+        return;
+    }
+
+    string [int] description;
+    string url = "inv_use.php?pwd=" + my_hash() + "&which=3&whichitem=11116";
```

Comment:
is the hash actually needed on this use link? i suspect it is just wondering if you tested without it.

Reply:
Yep, it doesn't work without the hash.  If you open misc inventory in its own window and then find the link to use the certificate, it points here (with the hash).